minimal_redux_poc/src/reducers/manifests.js

```diff
@@ -14,7 +15,7 @@ const manifestsReducer = (state = {}, action) => {
     case ActionTypes.RECEIVE_MANIFEST:
       return Object.assign({}, state, {
         [action.manifestId]: {
-          json: action.manifestJson,
+          json: manifesto.create(action.manifestJson),
```

Comment:
So, I think that this is more than just the `json`, it's the Manifesto instance, right? Perhaps we should call the property something more accurate. I've been using `manifestation`, but maybe that's not intuitive. At any rate, we should be able to call any of the manifesto methods off that. I also think we are going to need to massage the data sometimes, so I like the idea of being able to add a mixins or helpers file like I do [here](https://github.com/pulibrary/figgy/blob/61cf526269881f722dca72501f0a4945956396ea/app/javascript/store/vuex/actions.js#L11).

Reply:
Good point on this. I've gone ahead and updated the key name here. Perhaps we can do the mixins on an addon PR in the future?